######################### Ground_filter ######################
do_ground_filtering: true
clip_height: 0.4
sensor_height: 0
min_distance: 2.0
max_distance: 75.0
num_iter: 3
num_lpr: 20
th_seeds: 1.2
th_dist: 0.3

######################### Map_genrator ######################
seq_data_root: /home/yyj/download/kaist/
current_seq: urban17 #后面还需要加 / 
submap_frame_num: 50 #调用最新关键帧的前多少帧组成submap
submap_cube_size: 30 # submap cube的 完整边长 
submap_gap: 15
submap_point_num: 10000





